megamek/src/megamek/client/ui/swing/UnitEditorDialog.java

```diff
@@ -955,9 +967,162 @@ private void setupAeroSystemPanel() {
             gridBagConstraints.weightx = 1.0;
             panSystem.add(rightThrusterCrit, gridBagConstraints);
         }
+        
+        if (aero instanceof Jumpship) {
+            Jumpship js = (Jumpship) aero;
+            //Grav Decks
+            gridBagConstraints.gridx = 0;
+            gridBagConstraints.gridy++;
+            gridBagConstraints.weightx = 0.0;
+            panSystem.add(new JLabel("<html><b>" + "Grav Decks"
```

Comment:
I'm sure the rest of this file isn't up to "code", so to speak, but it would be nice if the text like `"Grav Decks"` (etc) were in resources. The HTML would not though.

Reply:
Resources does not appear to be used at all by MM. I'd feel better setting this up for the whole file in a separate PR.

Reply:
This is done.  I also updated Aero, Dropship and Jumpship where I had getCritDamageString()